jobbergate-agent/Dockerfile.dev

```diff
@@ -1,13 +1,13 @@
-FROM python:3.12-slim-bookworm
+FROM slurm-docker-cluster
 
 WORKDIR /app
 
-RUN apt update && apt install -y curl libpq-dev gcc
+RUN apt update && apt install -y curl libpq-dev gcc python3-dev python3-pip htop && \
+    ln -s /usr/bin/python3 /usr/bin/python
 
 RUN curl -sSL  https://install.python-poetry.org | \
     POETRY_HOME=/opt/poetry POETRY_VERSION=1.5.1 python && \
-    cd /usr/local/bin && \
-    ln -s /opt/poetry/bin/poetry && \
+    ln -s /opt/poetry/bin/poetry /usr/local/bin && \
```

Comment:
It includes no change in functionality. The `cd` was confusing, so I included the target path instead of changing directories. Since `/usr/local/bin` is a directory, it, in fact, creates the link inside with the same filename `/usr/local/bin/poetry`. I'll follow your suggestion because it makes this line cleaner.

Reply:
solved in [19be649](https://github.com/omnivector-solutions/jobbergate/pull/498/commits/19be6495094d96b2ede2b7900dede98fc500000b)